saneterm/color.py

```diff
@@ -0,0 +1,158 @@
+from enum import Enum, auto, unique
+
+from gi.repository import Gdk
+
+@unique
+class BasicColor(Enum):
+    BLACK = 0
+    RED = 1
+    GREEN = 2
+    YELLOW = 3
+    BLUE = 4
+    MAGENTA = 5
+    CYAN = 6
+    WHITE = 7
+
+# colors are (almost) the same as XTerm's default ones,
+# see X11's rgb.txt for the RGB values
```

Comment:
Maybe add a link, i.e. where do I find this file? Maybe just reference https://en.wikipedia.org/wiki/X11_color_names instead?